lib/cisco_node_utils/cmd_ref/interface_ospf.yaml

```diff
@@ -6,6 +6,11 @@ _template:
   get_command: 'show running interface all'
   context: ["interface %s"]
 
+all_interfaces:
```

Comment:
Do we need this? We have the same property in interface.yaml (all_interfaces). can it be reused here?

Reply:
update: seems like you need it as you changed the object code to call the new yaml properties. what is the reason here? The yaml seems pretty much the same, may be something is missing. could you add a comment?

Reply:
There is a pretty big difference now.  The `self.all_interfaces` method in `interface.rb` uses the `show running-config interface | section '^interface'` command along with a filter `get_value: '/(.*)/'` that gets all of the interfaces and associated interface data.  We need all of the data so that we can keep track of all interfaces and weather or not the interface is in a default state.

For the other objects `interface_portchannel.rb` and `interface_ospf.rb` we only need a simple list of interface names and not the additional data under each interface.